dm/helpers/google_netblock_ip_ranges/README.md

```diff
@@ -0,0 +1,56 @@
+# Important Google IP ranges helper
+
+This helper creates firewall template rules for a network with Google important ranges.
+
+## Prerequisites
+
+- Install [gcloud](https://cloud.google.com/sdk)
+- Create a [GCP project, set up billing, enable requisite APIs](../project/README.md)
+- Create a [network](../network/README.md)
+- Grant the [compute.networkAdmin or compute.securityAdmin](https://cloud.google.com/compute/docs/access/iam) IAM role to the project service account
+
+## Deployment
+
+### Resources
+
+- [compute.beta.firewall](https://cloud.google.com/compute/docs/reference/rest/beta/firewalls)
+  
+  `Note:` The beta API supports the firewall log feature.
+
+### Usage
+
+1. Clone the [Deployment Manager samples repository](https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit):
+
+```shell
+    git clone https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit
```

Comment:
Missing `cd cloud-foundation-toolkit` ( this is a common issue with most of our Readmes, but lets fix it.

Reply:
Added `cd cloud-foundation-toolkit`